code/modules/antagonists/bloodsucker/datum_bloodsucker.dm

```diff
@@ -252,12 +253,26 @@
 		owner.hasSoul = TRUE
 //owner.current.hellbound = FALSE
 
-/datum/antagonist/bloodsucker/proc/RankUp()
+/datum/antagonist/bloodsucker/proc/ForcedRankUp() //Big ol SKYRAT EDIT
 	set waitfor = FALSE
 	if(!owner || !owner.current)
 		return
 	bloodsucker_level_unspent ++
 	// Spend Rank Immediately?
+	if(istype(owner.current.loc, /obj/structure/closet/crate/coffin))
+		SpendRank()
+	else
+		to_chat(owner, "<EM><span class='notice'>You have forced your powers to further through the power of blood; Sleep within your lair to claim your boon.</span></EM>")
+		if(bloodsucker_level_unspent >= 2)
+			to_chat(owner, "<span class='announce'>Bloodsucker Tip: If you cannot find or steal a coffin to use, you can build one from wooden planks.</span><br>")
+
+
+/datum/antagonist/bloodsucker/proc/RankUp() //Adjusted due to nighttime changes.
+	set waitfor = FALSE
+	if(!owner || !owner.current)
+		return
+	bloodsucker_level_unspent += 5 //5x the levels
```

Comment:
5? With the ability to purchase levels too?

Reply:
5 per hour

Reply:
Lets do some math here @Azarak 

To purchase 5 level ups, thats 200 blood + 100/200/350/400 blood per finalisation
So in total you'll spend 1250 blood levelling yourself up to just _level five_. This is because the cost itself to level up increases; along with the cost to 'finalise' that level in a coffin.

Reply:
_This means that its unviable from a gameplay perspective to level yourself up rapidly; since every level up will cost ~70-80% of your total blood (At level 1 you only have 600 blood, meaning it'll cost ~350 blood), from there, you'll have 750 capacity, with a ~500-600 cost. This means its mechanically unviable to level up twice in a row without having to get more blood_